code/datums/craft/recipe.dm

```diff
@@ -22,7 +26,13 @@
 /datum/craft_recipe/proc/spawn_result(obj/item/craft/C, mob/living/user)
 	var/atom/movable/M = new result(get_turf(C))
 	M.Created(user)
-	M.dir = user.dir
+	switch (C.recipe.dir_type)
+		if (0)  // spawn the result in the user's direction
```

Comment:
Please use defines

Reply:
Done. Wasn't very inspired for the name of the constants...